symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/exception/NoBotConfigException.java

```diff
@@ -0,0 +1,14 @@
+package com.symphony.bdk.core.config.exception;
+
+import org.apiguardian.api.API;
+
+/**
+ * Thrown when the bot configuration is not specified. The BDK will only be runnable in OBO mode.
+ */
+@API(status = API.Status.STABLE)
+public class NoBotConfigException extends RuntimeException {
+
+  public NoBotConfigException(String message) {
```

Comment:
Since you have a single error message (`NO_BOT_CONFIG_MESSAGE `) you could put it directly in the exception, and always `throw new NoBotConfigException ()`